etc/settings.json

```diff
@@ -0,0 +1,5 @@
+    {
```

Comment:
this is the settings.json file and contains secrets it should not be pushed on github. 
Now the secrets are exposed